### PR TITLE
implementation for storing chat data and retrieving it from db

### PR DESCRIPTION
**Pull request by Bakushin10**
implement server/client side to store chat JSON object to db.
when the page is loaded, it will retrieve the chat data from db and display it. 

No UI is edited yet.

### CODE REVIEW

**Comment by pryanalewis** [comment]
Hey Shin! So I can see the chat history the only thing is when you are in the chat and you type a message it shows it above all the old messages, then when you leave and come back to the chat it moves the message down to the bottom.

**Review by pryanalewis — COMMENTED** [review]
Hey Shin! So I can see the chat history the only thing is when you are in the chat and you type a message it shows it above all the old messages, then when you leave and come back to the chat it moves the message down to the bottom.

**Comment by yotoo2920** [comment]
@Bakushin10 Pryana is right about the order, when retrieving the data from the db the message are in the right order, but when chatting the most recent text is going to the top instead of at the bottom of the last message

File: modules/myclasses/scripts/generate-ssl-certs.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	echo "i.e: bash scripts/generate-ssl-certs.sh"
+	exit -1
+fi
+
+echo "Generating self-signed certificates..."
+mkdir -p ./config/sslcerts
+openssl genrsa -out ./config/sslcerts/key.pem 1024
+openssl req -new -key ./config/sslcerts/key.pem -out ./config/sslcerts/csr.pem
+openssl x509 -req -days 9999 -in ./config/sslcerts/csr.pem -signkey ./config/sslcerts/key.pem -out ./config/sslcerts/cert.pem
+rm ./config/sslcerts/csr.pem
```
Comment (yotoo2920) [inline modules/myclasses/scripts/generate-ssl-certs.sh:16]: @Bakushin10 why do we need this generate-ssl-certs.sh or was it auto-generated ?

**Comment by Bakushin10** [comment]
@yotoo2920 @pryanalewis  I am aware of the problem. But I think it is a front end problem and should be an easy fix. I am thinking of fixing the issues along with the UI changes.

**Comment by yotoo2920** [comment]
I just checked the order the message are being added in the db and it look correct!!!
therefore, it is a just a matter how there are being displayed on the client side.

**Review by yotoo2920 — APPROVED** [review]
Accepting pull request since we already are in the same page about messages order

**Review by tjsparks5 — APPROVED** [review]
I encountered the same issue that Pryana and Dario mentioned. But apart from that everything looks great! I'd try to fix the issue before pulling to master though.

**Pull request merged**